run.js

```diff
@@ -0,0 +1,15 @@
+var path = require('path');
```

Comment:
Can you add a comment to this one explaining what it is (i.e., that it is for testing)

Reply:
I suspect that the 'path' module is not necessary.  Nothing following is using that.